spec/requests/dashboard_request_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Account::DashboardController, type: :request do
+  context "when the user is signed in" do
+    describe "GET :index" do
+      include_context :authorize_admin
+
+      it "returns a success response" do
+        get account_root_path
```

Comment:
я ж написав нижче, що має бути логічне розділення коду

```suggestion
        get account_root_path

```

Reply:
так само і в інших тестах

Reply:
зробила